### DISCUSSION TITLE
GPSR images

### DISCUSSION THREAD

**Discussion by dansharpe83** [amzn/selling-partner-api-models#4208] [General]
with the new gpsr tags being required, is there any plan to add image tags to the schemas? 
we can see they are in the csv upload templates under other_image_url_ps01.
when are they being added to the schemas?

**Comment by jawheral** [comment]
Hi @dansharpe83,

Compliance images: PS01-PS06 are available in the schemas as "image_locator_ps01", "image_locator_ps02",...
If these are not available in your schema, could you please provide the ProductType & Marketplace?

Best,
Jawher
Selling Partner API Solutions Architect

**Reply by dansharpe83** [reply]
Hi @jawheral ,

i just went and got the latest SHIRT schema for UK marketplace (A1F83G8C2ARO7P), and i cant see any attributes for it:
![image](https://github.com/user-attachments/assets/485280d0-343d-4f47-bfa9-974a2c7295ed)

i've also checked a few other product types (APPAREL_GLOVES, SLEEPING_BAG) and cant see these attributes anywhere in the schema files

**Reply by jawheral** [reply]
HI @dansharpe83,

The attributes are however available on other EU marketplaces such as Germany, France, Italy,... for the same product type. Are you sure clothing products require GPSR compliance in the UK? if not, that could be the reason why these attributes are not available there. 
There is also a possibility that these attributes are still not added to product types in the UK marketplace and will eventually be added.

Best,
Jawher
Selling Partner API Solutions Architect

**Comment by ritikacd** [comment]
Hi @dansharpe83 

Please note that you are not required to provide GPSR compliance document in UK marketplace, however you would need to provide those for other EU marketplaces. Listings APIs have a new attribute `compliance_media` that can be used to provide these documents. To comply with GPSR support, you need to submit following attributes in your listings submissions: 
- gpsr_manufacturer_email_address
- dsa_responsible_party_address
- gpsr_safety_attestation
- compliance_media

Call Product Type Definitions API to see the schemas requirement for the attributes mentioned above. We will be sending an update shortly on adding GPSR attributes using Listings API, make sure to check [announcements](https://developer-docs.amazon.com/sp-api/changelog) page. 

Thanks,
Ritika
Selling Partner API Solutions Architect
